src/components/seekbar/seekbar.js

```diff
@@ -542,7 +542,7 @@ class SeekBar extends Component {
    */
   render(props: any, state: Object): React$Element<any> {
     const virtualProgressWidth = `${(props.virtualTime / props.duration) * 100}%`;
-    const progressWidth = `${(props.currentTime / props.duration) * 100}%`;
+    const progressWidth = `${props.forceFullProgress ? 100 : (props.currentTime / props.duration) * 100}%`;
```

Comment:
why is needed?

Reply:
Now the live time is linear and duration is always bigger than the current time. For that reason you will see unfulfilled progress on the progress bar's edge. When we are on live edge this has no meaning so we would like to see full progress.